package.json

```diff
@@ -14,8 +14,8 @@
     "icons": "npx @svgr/cli --title-prop --no-dimensions --replace-attr-values \"#000=currentColor\" -d src/components/icons src/components/icons/svg && eslint ./src/components/icons --fix",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "test": "npx concurrently --kill-others -s first \"yarn dev\" \"cypress run\"",
```

Comment:
You might want to specify a browser, right not the tests are running on `electron` when I use `yarn test`